### PR TITLE
Planimal pics amaar

### PR DESCRIPTION
**Pull request by AmaarM**

### CODE REVIEW

File: Flora-Fauna-Frontend/src/components/LandingPage/LandingPage.jsx [hunk 1]
```diff
@@ -15,7 +15,6 @@ export default function LandingPage() {
    //Sends input value to be requested, then navigates to search page, which will display the results
    function handleOnSubmit(){
       //Send search request
-      console.log("hello")
```
Comment (LifeBringer) [inline Flora-Fauna-Frontend/src/components/LandingPage/LandingPage.jsx:18]: Comment out

File: Flora-Fauna-Frontend/src/contexts/search.jsx [hunk 2]
```diff
@@ -1,40 +1,63 @@
...
     setIsLoading(true);
     setInitial(false);
-    try{
-      const req = await ApiClient.searchResults({data:data})
+    try {
+      const req = await ApiClient.searchResults({ data: data });
       setSearchResults(req);
-    }
-    catch(err){
-      console.log(err)
```
Comment (LifeBringer) [inline Flora-Fauna-Frontend/src/contexts/search.jsx:24]: Remove

**Review by LifeBringer — CHANGES_REQUESTED** [review]
Remove all console.logs

**Comment by LifeBringer** [comment]
(or comment it out)

File: Flora-Fauna-Backend/routes/planimal.js [hunk 3]
```diff
@@ -12,4 +12,15 @@ router.post("/search", async(req,res,next) => {
     }
 })
 
+//Route that returns pictures of request
+router.post("/search/getPhotos", async(req,res,next) => {
```
Comment (ericloufoua) [inline Flora-Fauna-Backend/routes/planimal.js:16]: RECOMMENDED: "/search/photos" better conforms to naming convention.

File: Flora-Fauna-Frontend/src/components/LandingPage/LandingPage.jsx (hunk 1 above)
Comment (ericloufoua) [reply Flora-Fauna-Frontend/src/components/LandingPage/LandingPage.jsx:18]: RECOMMENDED: Use Javascript Console Utils for making and cleaning up your console log statements.

**Review by ericloufoua — APPROVED** [review]
LGTM

**Pull request merged**